### DISCUSSION TITLE
Fix sort by star rating in song select

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#18397] [Q&A] [answered]
I know its duplicated and asked multiple time but is there something new planned with the filter by star rating that is fully broken in song select ?
The last time I asked Realm implementation was in progress but what about now ? Sort by star rating is one of the most used filter in stable and it's really frustrating to not be able to use it in lazer.

**Comment by frenzibyte** [comment, marked as answer]
What do you mean by "fully broken in song select", do you mean that lazer doesn't split beatmap difficulties in a beatmap for sorting to become fully based on the star rating of each one? if so, this is dependant on https://github.com/ppy/osu/issues/2547.

**Reply by Biokax** [reply]
> What do you mean by "fully broken in song select", do you mean that lazer doesn't split beatmap difficulties in a beatmap for sorting to become fully based on the star rating of each one? if so, this is dependant on #2547.

I mean it does not match stable at all, actually sorting by star rating is useless because its sort only in the mapset and its not expected. Sort "fully based on the star rating of each one" is better, from https://github.com/ppy/osu/issues/2547 it seems to be part of the future song select redesign, but nothing new here since september this is why I made this thread in order to have fresh news on this part

**Reply by frenzibyte** [reply]
There are no fresh news, all open issue threads are updated accordingly, so there's no need to open new threads for that purpose.

I can see what you mean now, and that is indeed a duplicate of #2547. It may likely be resolved with the new design of the song select, which will be worked on soon.
